api/diagnosis/diagnosis_routes.py

```
@@ -0,0 +1,88 @@
+        frames = await get_frames_from_queue(device_uid)
+        if not frames:
+            raise ErrorForm(404, "no_frames", "get_diagnosis_result(프레임 조회)")
+
+        input_array = preprocess_input_data(frames)
+    except ErrorForm as e:
+        logger.error(f"error_code:{e.code}, {e.message}, get_diagnosis_result(입력 데이터 처리), {e.detail_message}")
+        return create_error_response(e.code, e.message, "get_diagnosis_result(입력 데이터 처리)", e.detail_message)
+
+    try:
+        predicted_class  = model.predict(input_array)
+        logger.info(f"모델 결과 반환 값(확률): {predicted_class}")
```

Ruff (0.11.9)

Logging statement uses f-string

(G004)